File: benchmarks/accuracy/fixtures/java/control_interprocedural_parameterized_sql.java
import java.sql.*;
import javax.servlet.http.*;

public class UserController extends HttpServlet {
    private Connection conn;

    private ResultSet findUser(String name) throws SQLException {
        PreparedStatement stmt = conn.prepareStatement("SELECT * FROM users WHERE name = ?");
        stmt.setString(1, name);
        return stmt.executeQuery();
    }

    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws SQLException {
        String name = request.getParameter("name");
        findUser(name);
    }
}

File: benchmarks/accuracy/fixtures/go/control_interprocedural_parameterized_sql.go
package main

import (
	"database/sql"
	"net/http"
)

var db *sql.DB

func findUser(name string) (*sql.Rows, error) {
	return db.Query("SELECT * FROM users WHERE name = $1", name)
}

func handler(w http.ResponseWriter, r *http.Request) {
	name := r.URL.Query().Get("name")
	findUser(name)
}
